src/role-access/components/answers/answers.component.html

```diff
@@ -0,0 +1,21 @@
+<div>
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l">{{caption}}</span>
+    {{heading}}
+  </h1>
+  <span class="govuk-hint">{{hint}}</span>
+  <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+    <div class="govuk-summary-list__row" *ngFor="let answer of answers">
+      <dt class="govuk-summary-list__key">
+        {{answer.label}}
+      </dt>
+      <dd class="govuk-summary-list__value" [innerText]="answer.value">
+      </dd>
+      <dd class="govuk-summary-list__actions">
+        <a class="govuk-link" routerLink="." (click)="changeAction(answer.action)">
+          Change<span class="govuk-visually-hidden"> who the exclusion is for</span>
```

Comment:
@rayliangatos - I might be wrong but I think the space is intentional, so that it reads "Change who the exclusion is for" but the ` who the exclusion is for` part can be hidden.

Reply:
Hi @rayliangatos. It's per GDS guidelines for accessibility. If you view the source of the summary list on this page you will see I coded it as expected (admittedly it looks a bit odd, but it's hidden from view and screen readers would pick up the space) https://design-system.service.gov.uk/components/summary-list/ .

Reply:
thanks addressing the issue and let me know this side case.